org/all-prs.ts

```diff
@@ -1,9 +1,25 @@
-import { schedule } from "danger"
+import { schedule, danger, warn, fail } from "danger"
 
-// Check for dependency changes
+const pr = danger.github.pr
+const bodyAndTitle = (pr.body + pr.title).toLowerCase()
+
+// Custom modifiers for people submitting PRs to be able to say "skip this"
+const wipPR = bodyAndTitle.includes("wip ") || bodyAndTitle.includes("[wip]")
+
+// Highlight package dependencies on Node projects.
 import yarn from "danger-plugin-yarn"
 schedule(yarn())
 
-// Check out docs
+// Keep our Markdown documents awesome
 import spellcheck from "danger-plugin-spellcheck"
 schedule(spellcheck({ settings: "artsy/artsy-danger@spellcheck.json" }))
+
+//  RFC: #5 - No PR is too small to warrant a paragraph or two of summary
+if (pr.body.length === 0) {
+  fail("Please add a description to your PR.")
+}
+
+//  RFC: #13 - Always ensure we assign someone, so that our Slackbot work correctly
+if (!wipPR && pr.assignee === null) {
+  warn("Please assign someone to merge this PR, and optionally include people who should review.")
```

Comment:
I know this is how Artsy currently uses reviews/assigns and it's tied to the slackbot, but it drives me crazy because it seems exactly backwards. Reviews can actually fail a PR, so they shouldn't be optional IMHO.

/cc @alloy

Reply:
It’s tied more to the fact that [GitHub’s API](https://developer.github.com/v3/issues/#list-issues) allows you to see who is assigned to an issue but not _yet_ who has been asked to review a PR ¯\_(ツ)_/¯

Having said that, since then I realised that _maybe_ GitHub’s search API would be able to give us this information, just haven’t dug in yet. Wanna take a stab? We need a single feed of all PRs in the org that include info about who have been requested reviews of.

Finally, while I agree that we want the bot to be able to report requested reviews, I don’t agree that it’s backwards. The one thing that is _not_ optional ever is an assignee imo, otherwise who is going to do the merge?

Reply:
I just took 10m on this:

* The GraphQL API has nothing when diving through the org, or the user
* The REST Search only includes assignee: https://developer.github.com/v3/search/#search-issues

Reply:
Bummer.

FYI I asked GitHub to add support for it immediately after they released the ‘request review’ feature, so there’s not much else we can do but wait.